README.old.md

```diff
@@ -0,0 +1,2 @@
+# hello-react-front-end
+This projects aims to set react as a separate app from rails
```

Comment:
- [ ] As requested by the previous reviewer, Please make sure to remove this old README file. It will make the repo more professional.